BUILD.gn

```diff
@@ -31,9 +31,7 @@ config("config") {
     defines += [ "LEGACY_FUCHSIA_EMBEDDER" ]
   }
 
-  if (is_debug) {
-    defines += [ "FLUTTER_ENABLE_DIFF_CONTEXT" ]
-  }
+  defines += [ "FLUTTER_ENABLE_DIFF_CONTEXT" ]
```

Comment:
Do we have any data on how this affects benchmarks?

Do we want to make this something guarded by a flag/setting until we have that data?

Reply:
Just to be clear. This will affect whether the code compiles in or not.

There should be a runtime flag that should govern whether we use the mechanism or not - so that developers and customers can try running with and without the flag to see if they gain performance without having to do a full engine rebuild.

With the second runtime flag, this particular change in `defines` will only affect binary size and maybe a boolean test per frame.

Reply:
My preference would be to get this reviewed and landed and get data from our CI and internal benchmarks. If there are regressions, we can revert and reland under a flag.